pkg/hardwareutils/bmc/irmc.go

```diff
@@ -4,6 +4,11 @@ import (
 	"net/url"
 )
 
+const (
+	False = "False"
+	True  = "True"
```

Comment:
This is a bit dangerous since it is so easy to assume that they are booleans. Perhaps we could make it a bit clearer with a suffix?
```suggestion
	FalseStr = "False"
	TrueStr  = "True"
```

Reply:
Good point, done